### DISCUSSION TITLE
Allow linking to hidden pages from Persona Bar

### DISCUSSION THREAD

**Discussion by stephen-lim** [dnnsoftware/Dnn.Platform#3779] [General] [answered]
## Description of problem
We have a 3rd party module that has a complex administration module that would benefit from having a link in the Persona Bar (similar to the **Site Assets** link). Currently, we can create a link in the Persona Bar using SQL but it requires the page to be visible on the site menu. If the page is not visible on the menu, the link does not show up in the Persona Bar. We don't want to have a link on the site menu and Persona Bar as it would defeat the purpose. An extra link on the site menu takes up a considerable horizontal space.

## Description of solution
Allow Persona Bar to link to pages that are hidden from the menu.

## Description of alternatives considered
We currently have no choice but to make the page visible on the menu so that it shows up in the Persona Bar. 

Please note the **Site Assets** link is visible even though it does not appear on the menu is because this page is located in the special legacy **Admin** path. 

**http://site/Admin/File-Management**

However, as I understand this workaround path will be removed in the near future once the **Site Assets** module is migrated into the Persona Bar. Furthermore, the **Admin** path is a hidden path that is not readily accessible by users from the **Content > Pages** section of the Persona Bar making it difficult for users to create a new link.

**Comment by valadas** [comment, marked as answer]
I cannot find the other issue now but this was discussed in another issue. There was mixed opinions if this should be allowed or not. And also nobody actually submitted a pull request for any possible implementation of this. In the spirit of keeping the backlog under control, I am gonna move this to discussions until and if someone wants to contribute such a feature.
